recipes/grpc/all/conanfile.py

```diff
@@ -174,6 +174,11 @@ def generate(self):
         tc.cache_variables["gRPC_BUILD_GRPC_PYTHON_PLUGIN"] = self.options.python_plugin
         tc.cache_variables["gRPC_BUILD_GRPC_RUBY_PLUGIN"] = self.options.ruby_plugin
 
+        # Never download unnecessary archives
+        # (supported in gRPC >= 1.62.0)
+        tc.cache_variables["gRPC_DOWNLOAD_ARCHIVES"] = False
```

Comment:
I'm not particularly familiar with `gRPC` - these archives _aren't_ required? I agree that a conan recipe should never attempt to download external files, but if they are required then they should be added to `conandata.yml` and downloaded/extracted in the `source()` method.

Reply:
Based on the issue where the flag was introduced, it seems that they're not required for anything directly related to building or testing gRPC
https://github.com/grpc/grpc/pull/34587#issuecomment-1779337256

In addition, for all but one of the archives, no attempt is made to download the archives -- the CMake script checks for the presence of directories that are (in most cases) already present in the Conan source checkout.